### PR TITLE
CorfuStore Sample App

### PR DESCRIPTION
**Pull request by zalokhan**
## Overview

Description:
Sample application to write and read from CorfuStore.

Why should this be merged: 
Sample code to help write applications.

## Checklist (Definition of Done):

- [x] There are no TODOs left in the code
- [ ] [Coding conventions](https://github.com/CorfuDB/CorfuDB/wiki/Corfu-Style-Guidelines) (e.g. for logging, unit tests) have been followed
- [ ] Change is covered by automated tests
- [x] Public API has Javadoc

### CODE REVIEW

File: samples/proto/person_profile.proto [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import "corfu_options.proto";
+import "work_experience.proto";
+
+message Name {
+    string first_name = 1 [(org.corfudb.runtime.schema).secondary_key = true];
+    string last_name = 2 [(org.corfudb.runtime.schema).secondary_key = true];
+}
+
+message Person {
+    string first_name = 1 [(org.corfudb.runtime.schema).secondary_key = true];
```
Comment (lixinchengdu) [inline samples/proto/person_profile.proto]: Why first_name and last name are not replaced by Name?

File: samples/src/main/java/org/corfudb/samples/CorfuStoreSample.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+import samples.protobuf.PersonProfile.Name;
+import samples.protobuf.Vehicle.Car;
+
+import java.util.Map;
+
+/**
+ * Created by zlokhandwala on 10/12/19.
+ */
+public class CorfuStoreSample {
+    private static final String USAGE = "Usage: HelloCorfu [-c <conf>]\n"
```
Comment (lixinchengdu) [inline samples/src/main/java/org/corfudb/samples/CorfuStoreSample.java]: HelloCorfu -> CorfuStoreSample or other name?

File: samples/src/main/java/org/corfudb/samples/CorfuStoreSample.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+        /**
+         * First, the application needs to instantiate a CorfuRuntime,
+         * which is a Java object that contains all of the Corfu utilities exposed to applications.
+         */
+        CorfuRuntime runtime = getRuntimeAndConnect(corfuConfigurationString);
+
+        CorfuStore corfuStore = new CorfuStore(runtime);
+
+        String namespace = "aaa";
```
Comment (lixinchengdu) [inline samples/src/main/java/org/corfudb/samples/CorfuStoreSample.java]: need a more readable name

File: samples/proto/person_profile.proto [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    string last_name = 2 [(org.corfudb.runtime.schema).secondary_key = true];
+    int32 age = 3;
+    samples.protobuf.Experience exp = 4;
+}
```
Comment (vjeko) [inline samples/proto/person_profile.proto]: Newline.

File: samples/src/main/java/org/corfudb/samples/CorfuStoreSample.java [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+import org.docopt.Docopt;
+import samples.protobuf.PersonProfile.Name;
+import samples.protobuf.Vehicle.Car;
+
+import java.util.Map;
+
+/**
+ * Created by zlokhandwala on 10/12/19.
+ */
+public class CorfuStoreSample {
```
Comment (vjeko) [inline samples/src/main/java/org/corfudb/samples/CorfuStoreSample.java]: Comments.

File: samples/proto/person_profile.proto (hunk 1 above)
Comment (zalokhan) [reply samples/proto/person_profile.proto]: Done.

**Pull request merged**